ssts/utils/main_methods.py

```diff
@@ -0,0 +1,887 @@
+import os
```

Comment:
Main methods is too generic. We could break up this file into 3 smaller ones: Noise removal (outlier extraction and frequency removal), correlation and distributions methods, and plotting functions.